### ISSUE TITLE
[console] removing rolefrom user no longer updates ldap

### ISSUE THREAD

**Issue by landryb** [georchestra/georchestra#2409]
Testing lots of cases after #2108 - it seems on a user page, adding a (georchestra, other) role to a user correctly updates the ldap:
```
{"users":["uid"],"PUT":["MOD_APL"],"DELETE":[]}
``` 
the uid is present as a member in MOD_APL role as expected - but removing the role from the same user:
```
{"users":["uid"],"PUT":[],"DELETE":["MOD_APL"]}
``` 
the POST returns success, but the ldap is not updated, and the uid is still in the role.

**Comment by landryb** [comment]
@fvanderbiest/@cmangeat : can this one be looked at ? This is really impacting my current testing.....

**Comment by cmangeat** [comment]
Thank you for testing.
I think it has been fixed on https://github.com/georchestra/georchestra/pull/2408.

**Comment by landryb** [comment]
will pull #2408 locally and test, thanks!

**Comment by landryb** [comment]
This one is definitely fixed (pulled #2408 on top of a local branch spun off master to test) - thanks again.

tested:
  * adding a georchestra (or non-base) role to a user - works as before (log output from my code listening to ldap changes):
```
2019-01-31 10:38:37 DEBUG    ldapwatch ldapwatch     RECORD CHANGED:cn=GN_EDITOR,ou=roles,dc=craig,dc=fr
2019-01-31 10:38:37 INFO     ldapwatch autoshare     member list changed for role GN_EDITOR: added={'testuser'}, removed=set()
2019-01-31 10:39:35 DEBUG    ldapwatch ldapwatch     RECORD CHANGED:cn=MOD_APL,ou=roles,dc=craig,dc=fr
2019-01-31 10:39:35 INFO     ldapwatch autoshare     member list changed for role MOD_APL: added={'testuser'}, removed=set()
```
  * removing the  same role from the user - now works, ldap is correctly updated:
```
2019-01-31 10:36:57 DEBUG    ldapwatch ldapwatch     RECORD CHANGED:cn=MOD_APL,ou=roles,dc=craig,dc=fr
2019-01-31 10:36:57 INFO     ldapwatch autoshare     member list changed for role MOD_APL: added=set(), removed={'testuser'}
2019-01-31 10:38:34 DEBUG    ldapwatch ldapwatch     RECORD CHANGED:cn=GN_EDITOR,ou=roles,dc=craig,dc=fr
2019-01-31 10:38:34 INFO     ldapwatch autoshare     member list changed for role GN_EDITOR: added=set(), removed={'testuser'}
